models/comments-model.js

```diff
@@ -9,7 +9,31 @@ exports.fetchComments = (reqParams) => {
       [article_id]
     )
     .then((data) => {
-        console.log(data.rows)
+      return data.rows;
+    });
+};
+
+exports.postDbComment = (reqParams, reqBody) => {
+  const { article_id } = reqParams;
+  const { username, body } = reqBody;
+
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article Not Found" });
+      }
+    })
+    .then(() => {
+      if (typeof body !== "string") {
+        return Promise.reject({ status: 400, msg: "Comment must be text" });
+      }
```

Comment:
As far as I'm aware this is down to personal choice but if you wanted to I believe you could let your PSQL error handler deal with this instead of manually checking yourself :)